test/foundry-tests/reactors/RelayOrderReactorTest.t.sol

```diff
@@ -195,19 +192,41 @@ contract RelayOrderReactorTest is GasSnapshot, Test, PermitSignature, DeployPerm
         vm.warp(block.timestamp + 250);
 
         vm.expectEmit(true, true, true, true, address(reactor));
-        emit Fill(orderHash, address(filler), swapper, order.info.nonce);
+        emit Fill(order.hash(), address(filler), swapper, order.info.nonce);
         // expect we can execute the first order
         vm.prank(filler);
         reactor.execute(signedOrder, filler);
         assertEq(tokenIn.balanceOf(address(filler)), 250000000000000000);
 
-        // change deadline to sig and orderHash are different
-        order.info.deadline = block.timestamp + 1500;
-        // however nonce 0 will have been used
         signedOrder = SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
         // expect revert
         vm.prank(filler);
         vm.expectRevert(InvalidNonce.selector);
         reactor.execute(signedOrder, filler);
     }
+
+    function test_execute_reverts_EndTimeBeforeStartTime() public {
+        RelayOrder memory order = RelayOrderBuilder.initDefault(tokenIn, address(reactor), swapper).withStartTime(
+            block.timestamp + 20
+        ).withEndTime(block.timestamp + 10);
+
+        SignedOrder memory signedOrder =
+            SignedOrder(abi.encode(order), signOrder(swapperPrivateKey, address(permit2), order));
+
+        vm.prank(filler);
+        vm.expectRevert(ReactorErrors.EndTimeBeforeStartTime.selector);
+        reactor.execute(signedOrder, filler);
+    }
+
+    function test_execute_reverts_DeadlineBeforeEndTime() public {
+        RelayOrder memory order =
+            RelayOrderBuilder.initDefault(tokenIn, address(reactor), swapper).withEndTime(block.timestamp + 200);
```

Comment:
nit: can we explictly do .deadline(block.timestamp) here to show the test case?

Reply:
Yeah sure, i think the default is actually block.timestamp + 100 so I can make that explicit